### DISCUSSION TITLE
Unavailable icons

### DISCUSSION THREAD

**Discussion by esphoenixc** [tailwindlabs/heroicons#879] [Ideas]
Hi first, thanks for creating awesome icons.

While I was using heroicons, I needed to have mic-slash but there isn't one. There's video-camera-slash and videoc-camera icon which I am using it. So is there a way for me to draw or have the mic-slash icons? Not only mic, but also in general monitor and monitor-slash icons and so on.

**Comment by DistantMe** [comment]
I'll +1 this, was literally just looking for the exact same thing. Building an audio/video interface needing enable/disable (mute/unmute) buttons and saw that this one was missing on the microphone.

@adamwathan Do you know if there is any chance of new icons coming soon? This one would be an awesome addition if so =)

**Comment by florentmsl** [comment]
I think at least a `microphone-slash` would make sense, since `video-camera-slash` and `video-camera` are available

**Comment by ndri** [comment]
I added `microphone-slash` to [Sidekickicons](https://github.com/ndri/sidekickicons/), which are meant to be used as a companion to Heroicons. Browse them here: https://sidekickicons.com/?icon=microphone-slash
